lib/daru/view/adapters/highcharts/display.rb

```diff
@@ -3,7 +3,7 @@
 
 module LazyHighCharts
   def self.init_script(
-    dependent_js=['highstock.js', 'highcharts-more.js', 'modules/exporting.js',
+    dependent_js=['highstock.js', 'map.js', 'modules/exporting.js',
```

Comment:
Since it is constant we must place all the constant variable into one file, to make it loosely coupled. Accessing these constants into multiple files will be better.